docs/workers.md

```diff
@@ -319,6 +319,18 @@ stream_writers:
     events: event_persister1
 ```
 
+The `events` stream also experimentally supports having multiple writers, where
+work is sharded between them by room ID. Note that you *must* restart all
+instances when adding or removing event persisters. An example `stream_writers`
```

Comment:
```suggestion
work is sharded between them by room ID. Note that you *must* restart all
even persister instances when adding or removing event persisters. An example `stream_writers`
```

While a little repetitive, I originally wasn't sure whether this meant all event persisters or all workers.

Reply:
I do mean all instances though. Will try and reword

Reply:
Oh, just say "restart all worker instances" perhaps?